cmd/zrepl/mgmt_conn.c

```diff
@@ -642,8 +635,9 @@ process_message(uzfs_mgmt_conn_t *conn)
 		strncpy(zvol_name, payload, payload_size);
 		zvol_name[payload_size] = '\0';
 
-		if ((zinfo = uzfs_zinfo_lookup(zvol_name)) == NULL) {
-			fprintf(stderr, "Unknown zvol: %s\n", zvol_name);
+		if (((zinfo = uzfs_zinfo_lookup(zvol_name)) == NULL) ||
+		    (zinfo->mgmt_conn != conn)) {
+			ZREPL_ERRLOG("Unknown zvol: %s\n", zvol_name);
```

Comment:
let's avoid using ZREPL_ERRLOG in mgmt_conn.c file. syslog logging is going to be removed from zrepl in the future and I already cleaned mgmt_conn.c file from them. All messages here should be either error messages to stderr or debug messages using DBGCONN (which end up on stderr as well). This particular message should be fprintf(stderr, ...) since it is not a debug message.

Reply:
ok.. I will change it.. one reason I changed this is - I'm not seeing the logs coming to /var/log/syslog.. where will these logs goes?